tests/docker/ducktape-deps/kgo-verifier

```diff
@@ -2,6 +2,6 @@
 set -e
 git -C /opt clone https://github.com/redpanda-data/kgo-verifier.git
 cd /opt/kgo-verifier
-git reset --hard a4dff215149f8acb4fde8617f19adeabbfa69d44
+git reset --hard fab9660131cd4e6426eda6cec93ecfe1f5c0529b
```

Comment:
`fab9660` points to a git commit in unmerged PR https://github.com/redpanda-data/kgo-verifier/pull/58. it would be best to point to the merge commit after that PR merges to `main` to ensure the git lineage tracks back to latest of what exists on `main` branch